.travis.yml

```diff
@@ -91,8 +91,6 @@ jobs:
             <<: *STANDARD_TEST_JOB
             stage: Test
             php: 7.3
-            before_script:
-                - php php-cs-fixer fix --rules @PHP71Migration,@PHP71Migration:risky,blank_line_after_opening_tag -q || travis_terminate 1
```

Comment:
I had to remove this because it applied changes that don't actually comply with our CS rules so the subsequent fixer run failed:
```diff
-dirname(dirname(__DIR__))
+dirname(__DIR__,2)
```
Note the missing space between "," and "2".

See https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/594286444#L1047.

I'm not sure why this line was here. Also, should we fix `CombineNestedDirnameFixer`?

Reply:
>I'm not sure why this line was here. Also, should we fix CombineNestedDirnameFixer?

I think so, `CombineNestedDirnameFixer` list `should run before MethodArgumentSpaceFixer` so there definitely is a priority issue we should resolve.

Reply:
This is already covered by a priority test. The issue here is that `method_argument_space` is not part of the rules applied in the first run (the one I removed).

We could restore this line by either:
- making `CombineNestedDirnameFixer` always add the space without relying on a subsequent rule;
- merge default rules of the project with the one added explicitly here (so maybe the subsequent run would become redudant).

Reply:
I think the first option has most benefits for all. I also think this is a bug in the current fixer so could be part of the next patch release, WDYT?

Reply:
See #4892.